.ruby-version

```diff
@@ -0,0 +1 @@
+3.2
```

Comment:
I don't know what this does, but it doesn't seem related to the Android app portion.